src/libcore/num/mod.rs

```diff
@@ -187,13 +187,29 @@ macro_rules! int_impl {
      $sub_with_overflow:path,
      $mul_with_overflow:path) => {
         /// Returns the smallest value that can be represented by this integer type.
+        ///
+        /// # Examples
+        ///
+        /// Basic usage:
```

Comment:
I did it to keep file consistency. The RFC isn't merged yet so for now I prefer to make it like other doc comments in this file. However, I can update other if you want.